lib/interactor/organizer.rb

```diff
@@ -41,11 +41,13 @@ module ClassMethods
       #     include Interactor::Organizer
       #
       #     organize [InteractorThree, InteractorFour]
+      #     organize InteractorFive
       #   end
       #
       # Returns nothing.
       def organize(*interactors)
-        @organized = interactors.flatten
+        @organized ||= []
+        @organized += interactors.flatten
```

Comment:
Rather than using `+=`, we could [`concat`](https://ruby-doc.org/core-2.4.1/Array.html#method-i-concat) which won't duplicate the array in memory.